library/Zend/Db/Adapter/Pdo/Abstract.php

```diff
@@ -302,7 +302,15 @@ protected function _quote($value)
     protected function _beginTransaction()
     {
         $this->_connect();
-        $this->_connection->beginTransaction();
+        if (!$this->_connection->beginTransaction()) {
+            $message = 'Begin transaction failed';
+            // PDO returns no error message for now, but it could be changed in future
```

Comment:
PDO may not return any message

Reply:
upravím, pokud mi ukážeš nějaký reálný případ, kdy to chybu hodí :)